google_guest_agent/cloud_init_accounts_test.go

```diff
@@ -0,0 +1,150 @@
+// Please note that the code below is modified by YANDEX LLC
+
+package main
+
+import (
+	"github.com/GoogleCloudPlatform/guest-agent/metadata"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/guest-agent/utils"
```

Comment:
Recommended to group imports: first system packages, then 3rd party, then ours.
https://github.com/golang/tools/commit/ed69e84b1518b5857a9f4e01d1f9cefdcc45246e